apps/dashboard/src/app/App.tsx

```diff
@@ -175,6 +176,21 @@ export class App extends React.Component<IAppSetting, IAppState> {
               featureFlights={parseFeatureFlights(this.state.featureFlights)}
             />
           )}
+          {this.state.application === "modelAssessmentForecasting" && (
+            <ModelAssessmentForecasting
+              {...applications[this.state.application].datasets[
+                this.state.dataset
+              ]}
+              theme={themes[this.state.theme]}
+              language={Language[this.state.language]}
+              version={
+                applications[this.state.application].versions[
+                  this.state.version
+                ]
+              }
+              featureFlights={parseFeatureFlights(this.state.featureFlights)}
```

Comment:
I'm guessing you copied this from the lines above but we're calling parseFeatureFlights a lot; could we save as a local variable and use that instead ? or is there any reason that this.state.featureFlights is not the parsed featureFlights ?

Reply:
This is a complicated one 🙂 
If I remember correctly it would be easiest to represent it as a string[] everywhere, BUT this is one of the dropdowns in the test env so that we can try it out with or without flights (and also combinations thereof). That, in turn, means we need to be able to encode it in the URL, e.g., 
```
/modelAssessment/adultCensusIncomeData/1%20-%20light/en/1/flight1&flight2&flight3
```
So then at some point we have to parse it from this string to string[]. I think that part is intuitive, but you asked a great question: why not have it as string[] on the state? 
The answer is that the state consists of `IAppSetting`s and those defined as follows
```
export const routeKey = [
  "application",
  "dataset",
  "theme",
  "language",
  "version",
  "featureFlights"
] as const;
export type IAppSetting = {
  [key in typeof routeKey[number]]?: string;
};
```
In other words, we have no choice but to make it a string. We could, of course, have a duplicate and try to keep them in sync, but repeating the parse function a few times seems like a small price to pay in order not to have to keep them synced.

I may very well have missed something somewhere that makes this easily solvable. In that case, we should definitely adjust. Wdyt?